packages/core/android/src/test/kotlin/com/datadog/reactnative/DdLogsTest.kt

```diff
@@ -270,4 +280,131 @@ internal class DdLogsTest {
             attributes = mockContext.toHashMap()
         )
     }
+
+    @Test
+    fun `M not forward logs W SDK is not initialized`() {
+        // When
+        whenever(mockDatadog.isInitialized()) doReturn false
+        var newTestedLogs = DdLogs(mockReactContext, mockLogger, mockDatadog)
+        newTestedLogs.debug(fakeMessage, mockContext, mockPromise)
+        newTestedLogs.info(fakeMessage, mockContext, mockPromise)
+        newTestedLogs.warn(fakeMessage, mockContext, mockPromise)
+        newTestedLogs.error(fakeMessage, mockContext, mockPromise)
+        testedLogs.debugWithError(
+            fakeMessage,
+            fakeErrorKind,
+            fakeErrorMessage,
+            fakeStacktrace,
+            mockContext,
+            mockPromise
+        )
+        testedLogs.infoWithError(
+            fakeMessage,
+            fakeErrorKind,
+            fakeErrorMessage,
+            fakeStacktrace,
+            mockContext,
+            mockPromise
+        )
+        testedLogs.warnWithError(
+            fakeMessage,
+            fakeErrorKind,
+            fakeErrorMessage,
+            fakeStacktrace,
+            mockContext,
+            mockPromise
+        )
+        testedLogs.errorWithError(
+            fakeMessage,
+            fakeErrorKind,
+            fakeErrorMessage,
+            fakeStacktrace,
+            mockContext,
+            mockPromise
+        )
+
+        // Then
+        verifyZeroInteractions(mockLogger)
+        val throwableCaptor = argumentCaptor<Throwable>()
```

Comment:
here and below it is better to specify exact exception type

```suggestion
        val throwableCaptor = argumentCaptor<IllegalStateException>()
```